dokang/app.py

```diff
@@ -19,4 +19,9 @@ def make_app(global_settings, **settings):
     config.add_route('search', '/')
     config.add_view('.views.search', route_name='search', renderer='templates/search.pt')
 
+    dynamic_doc_set_dir = settings.get('dokang.uploaded_docs.dir')
```

Comment:
(minor) Could be renamed as "uploaded_docs_dir"? I think that "dynamic_doc_set_dir" comes from a previous version of the configuration, and the name does not make much sense anymore.